test/test_eager_transforms.py

```diff
@@ -279,6 +279,14 @@ def test_argnums(self, device):
         self.assertEqual(gx, y)
         self.assertEqual(gy, x)
 
+    def test_empty_grad(self, device):
+        def f(x):
+            return (x['a']**2.0).sum()
+        inps = ({'a':torch.randn(10) + 3, 'b':torch.randn(10)})
+        grads = grad(f)(inps)
+        self.assertNotEqual(grads['a'].sum(), 0.0)
+        self.assertEqual(grads['b'].sum(), 0.0)
+
```

Comment:
nit: test_empty_grad -> test_zero_grad. "empty" can refer to an "empty tensor", which is a tensor that does not have storage (because it has a zero-sized dimension somewhere).